routes/movies.routes.js

```diff
@@ -0,0 +1,47 @@
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Celebrity.model");
+const router = require("express").Router();
+
+
+
+router.get("/movies/create", (req, res, next) => {
+    console.log("in")
+    Celebrity.find()
+        .then(celebrityArray => {
+            res.render("movies/new-movie", { celebrity: celebrityArray })
+        })
+        .catch(err => {
+            console.log(err)
+            next(err)
+        })
+})
+
+router.post("/movies/", (req, res, next) => {
+    Movie.create({
```

Comment:
the route should be `/movies/create` here, and same in the form to create new movie